src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -115,6 +116,8 @@
             puppet-config (config->puppet-config config)
             puppetserver-config (HashMap.)
             env-registry (puppet-env/environment-registry)]
+        (when disable_i18n
+          (.put puppet-config "disable_i18n" "" ))
```

Comment:
Might be worth putting a comment here about why the value is an empty string, but not sure. The fact that this has to obey the Puppet CLI arg rules is kinda bizarre and not at all clear based on this section of the code.

Reply:
Should this be in `config->puppet-config`?